addons/mail/models/mail_channel.py

```diff
@@ -514,6 +609,11 @@ def _extract_moderation_values(self, message_type, **kwargs):
                 moderation_status = 'pending_moderation'
         return moderation_status, email
 
+    def _message_subscribe(self, partner_ids=None, subtype_ids=None, customer_ids=None):
+        """ Do not allow follower subscription on channels. Only members are
+        considered. """
+        return True
```

Comment:
I asked myself the same question. Or logging something ?

Reply:
Between the two definitely raise. Logging is just adding noise in the console for what is now ultimately a user error (or programming error in case of custo or whatever), I don't think people managing servers would like a log for this.

Reply:
Allez, let us go for a raise (contrary to Odoo, hahaha ... such joker).